packages/edit-site/src/components/header/index.js

```diff
@@ -109,22 +114,25 @@ export default function Header( { openEntitiesSavedStates } ) {
 			</div>
 
 			<div className="edit-site-header_center">
-				<DocumentActions
-					entityTitle={ title }
-					entityLabel={
-						templateType === 'wp_template'
-							? 'template'
-							: 'template part'
-					}
-				>
-					{ templateType === 'wp_template' &&
-						( ( { onClose } ) => (
+				{ 'wp_template' === templateType && (
+					<DocumentActions
+						entityTitle={ entityTitle }
+						entityLabel="wp_template"
```

Comment:
This should be human-readable since it's used in screen readers (like `view $entityLabel details`), so I think it should be "template" here and "template part" in the other one :)

Reply:
Good catch, I totally missed that! 🤦‍♂️